### PR TITLE
Coverage for vtk_ui

### PR DESCRIPTION
**Pull request by chaosphere2112**
This is a work in progress PR; I figured since I'm adding quite a lot of code (almost entirely tests), I should get this submitted sooner than later. Test data is at UV-CDAT/uvcdat-testdata#50

### CODE REVIEW

**Comment by chaosphere2112** [comment]
To save @aashish24 some heartburn later on down the road, I'm getting this PR into GitHub sooner than later as I work through adding test coverage to my UI code. Currently sitting at 65% (which you can see for yourself if you `pip install coverage` and run `python vtk_ui_coverage` in the `testing/vcs/vtk_ui directory`).

**Comment by chaosphere2112** [comment]
Test coverage is at 71%, coverage of stuff in use is getting quite good; hit 99% on Textbox today.

**Comment by aashish24** [comment]
@chaosphere2112 this is great. Yes, please smaller the changes is easier to get the review done.

File: testing/vcs/vtk_ui/vtk_ui_coverage.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    module_file = os.path.basename(test)
+    module_name, ext = os.path.splitext(module_file)
+    print "Test", module_name
+    module = __import__(module_name)
+    test_class = getattr(module, module_name)
+    test_object = test_class()
+    test_object.do_test()
+
+c.stop()
+c.html_report(directory="covhtml")
```
Comment (aashish24) [inline testing/vcs/vtk_ui/vtk_ui_coverage.py]: Missing new line.

**Comment by aashish24** [comment]
@chaosphere2112 in order for you to run buildbot on this, you should set your origin to original uvcdat repo and not your fork. Let me know if you need specific git instructions.

**Comment by doutriaux1** [comment]
@chaosphere2112 should we close this one now that you're working off the other one.

**Comment by chaosphere2112** [comment]
Yup! :+1:

**Pull request closed without merging**